extensions/agent/src/dialogs/operatorDialog.ts

```diff
@@ -81,9 +81,9 @@ export class OperatorDialog extends AgentDialog<OperatorData> {
 
 	private initializeGeneralTab() {
 		this.generalTab.registerContent(async view => {
-			this.nameTextBox = view.modelBuilder.inputBox()
-				.withProperties({ width: '100%' })
-				.component();
+
```

Comment:
@abist the `width` property doesn't appear to work in this context.  For me, it makes the text box read-only.

Reply:
Not sure why, I'll take a look again